docs/sources/flow/config-language/syntax.md

```diff
@@ -35,9 +35,9 @@ The following example sets the `log_level` attribute to `"debug"`.
 log_level = "debug"
 ```
 
-All `<ATTRIBUTE_NAME>`s must be valid River [identifiers](#identifiers).
+All `ATTRIBUTE_NAME`s must be valid River [identifiers](#identifiers).
```

Comment:
Plurals with code elements are awkward.  (Can't find the google dev style guideline at the moment). I think we should write around this if possible as I did here.